app/views/dashboard/agency/agent_invitations/_actions.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="btn-group">
```

Comment:
would love to see this whole employee invitation process get refactored into one partial, used for both Wholesaler and Agency